UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -87,6 +88,24 @@ private void Update()
         movementVector.y = verticalMovement;
         characterController.Move(movementVector * Time.deltaTime);
 
+        if (characterController.isGrounded) {
```

Comment:
This method is getting pretty big.
Suggestion: put your code inside a function with a good name. This increase legibility among other developers.

Reply:
Sure. I will put it in another function.